sylvia/src/into_response.rs

```diff
@@ -0,0 +1,90 @@
+use cosmwasm_std::{CosmosMsg, Empty, Response, StdError, StdResult, SubMsg};
+
+/// Trait converting `SubMsg` to one carrying another chain-custom message
+pub trait IntoMsg<C> {
+    fn into_msg(self) -> StdResult<SubMsg<C>>;
+}
+
+/// `SubMsg<Empty>` can be made into any `SubMsg<C>`
+impl<C> IntoMsg<C> for SubMsg<Empty> {
+    fn into_msg(self) -> StdResult<SubMsg<C>> {
+        let msg = match self.msg {
+            CosmosMsg::Wasm(wasm) => CosmosMsg::Wasm(wasm),
+            CosmosMsg::Bank(bank) => CosmosMsg::Bank(bank),
+            CosmosMsg::Staking(staking) => CosmosMsg::Staking(staking),
+            CosmosMsg::Distribution(distribution) => CosmosMsg::Distribution(distribution),
+            CosmosMsg::Custom(_) => Err(StdError::generic_err(
+                "Custom Empty message should not be sent",
+            ))?,
+            #[cfg(feature = "stargate")]
+            CosmosMsg::Ibc(ibc) => CosmosMsg::Ibc(ibc),
+            #[cfg(feature = "stargate")]
+            CosmosMsg::Stargate { type_url, value } => CosmosMsg::Stargate { type_url, value },
+            _ => Err(StdError::generic_err(format!(
+                "Unknown message variant: {:?}",
+                self
+            )))?,
```

Comment:
First, this should never happen; it means we don't convert some variants. I know it is needed because of `non_exhaustive` on `CosmosMsg`, but its firing means: "There was an update on cw-std, and you didn't align Sylvia to that yet" - and this should be explicit in the message as it is confusing now. Something like "Unknownown message variant: {:?}. Please make sure you are using the up-to-date Sylvia version, and if so please issue a bug on the Sylvia repository".

This error basically always mean - Sylvia is behind cw-std.

Other thing - never do `Err(err)?` - clippy warns about that (here clippy sometimes gets lost). Just `return StdError::generic_error(...)` (+ `.into()` if conversion is needed, eg. you use anyhow, but here you return `StdError` anyway).